platform/admin/modules/admin-security-certificate-ui/src/main/webapp/js/view/UploadModal.view.js

```diff
@@ -76,14 +88,34 @@ define([
         regions: {
             fileuploadregion: '.file-upload-region',
             fileeditregion: '.file-edit-region',
-            errorregion: '.error-region'
+            errorregion: '.error-region',
+            urlcertregion: '.url-cert-region'
+        },
+        showUrlCerts : function(e) {
+            var view = this;
+            var url = this.$('.urlField').val();
+            var encodedUrl = btoa(url);
+            var model = new UrlCertificate.Response();
+            model.fetch({url: model.url + "/" + encodedUrl}).done(function() {
+                view.checkSave(e);
+            });
```

Comment:
We should show something on error.  Like if they give a malformed URL or the server does not response to https.